core/src/main/scala/kafka/coordinator/transaction/TransactionStateManager.scala

```diff
@@ -87,13 +85,16 @@ class TransactionStateManager(brokerId: Int,
   /** number of partitions for the transaction log topic */
   private val transactionTopicPartitionCount = getTransactionTopicPartitionCount
 
-  // this is best-effort expiration and hence not grabing the lock on metadata upon checking its state
+  // this is best-effort expiration and hence not grabbing the lock on metadata upon checking its state
```

Comment:
The original comment is for "the metadata object lock" actually, I will make it more clear.